### DISCUSSION TITLE
Grouping subscriptions

### DISCUSSION THREAD

**Discussion by kjurl** [libre-tube/LibreTube#2881] [Ideas]
Please do let me know if this already exists. A system for grouping subscriptions and being able to see grouped subscription feeds would be nice for the people who like to keep say _education_, _entertainment_ or _news_ etc ... channels separate. Great App

**Comment by Mehdi-YC** [comment]
yes this is the only think that stops me from switching from freetube , 
im a software dev , if someone can give me a ducumentation about how subscription works , i might be able to implement it

**Comment by trymeouteh** [comment]
I hope Piped implement this and if they do, this would be cool to see in Libretube.

https://github.com/TeamPiped/Piped/issues/490

**Reply by Bnyro** [reply]
It got added in #3431 and will be available in the next update, however without support from Piped's side.

**Reply by Mehdi-YC** [reply]
will it be a local subscription categories ?

**Reply by kjurl** [reply]
> It got added in #3431 and will be available in the next update, however without support from Piped's side.

any idea when the update will be available?

**Reply by Bnyro** [reply]
two days ago
